meson.build

```diff
@@ -1,6 +1,7 @@
-project('komorebi', 'vala', 'c',
+project('komorebi',
     version: '2.2.1',
-    meson_version: '>=0.49',
+    meson_version: '>=0.50',
+    default_options: ['prefix=/usr']
```

Comment:
I'm not into enforcing `prefix` to `/usr`. By default, users should install to `/usr/local`. Usually, `/usr` is for distro managed packages. There's also a reason for that behavior on mesonbuild.

Reply:
Oops; yeah, that wasn't supposed to be re-added, my bad :sweat_smile: . Omitting it should default `prefix` to `/usr/local` again, but there's no harm in making it explicit IMO.

Reply:
> Oops; yeah, that wasn't supposed to be re-added, my bad  . Omitting it should default `prefix` to `/usr/local` again, but there's no harm in making it explicit IMO.

In this case no. But hardcoding it makes your meson script not portable. Komorebi is only designed for posix-based systems so theres not big deal, but meson already does this for you in a portable way, so I dont see a reason for redefining it.

Reply:
Yeah, please don't hardcode it :blush:

Reply:
Sorry, I just realized I worded that in a bad way; I meant that I could remove this line again so it defaults to `/usr/local` again, but that there's probably no harm in changing this to `default_options: ['prefix=/usr/local']` instead;

Reply:
Fixed